apps/app/github-environment.tf

```diff
@@ -119,40 +39,17 @@ resource "github_branch_protection" "deployment" {
     dismiss_stale_reviews           = var.accesses_real_data ? true : var.app_config.branch.dismiss_stale_reviews
     restrict_dismissals             = var.accesses_real_data
     require_code_owner_reviews      = var.accesses_real_data
-    required_approving_review_count = max(var.app_config.branch.num_of_approvals, 1)
+    required_approving_review_count = var.app_config.branch.num_of_approvals
```

Comment:
don't agree that you should be able to deploy with 0 reviewers

Reply:
Or we could say `var.accesses_real_data ? max(var.num_of_approvals, 1) : var.num_of_approvals`

Reply:
Ok lets go with the latter then, I think an admin should have the ability to override if they deem appropriate